redux/app/components/app.tag

```diff
@@ -0,0 +1,38 @@
+<app>
+
+  <div class="app">
+    <h3>Riot with Redux</h3>
+
+    <p>
+        { this.store.getState().sampleReducer.name }
```

Comment:
Probably this should be somewhere else and the app should only receive the object it needs as option.